server/store/memory/peer_store_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016 The Chihaya Authors. All rights reserved.
+// Use of this source code is governed by the BSD 2-Clause license,
+// which can be found in the LICENSE file.
+
+package memory
+
+import (
+	"net"
+	"testing"
+
+	"github.com/chihaya/chihaya"
+	"github.com/chihaya/chihaya/server/store"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	hash1 = chihaya.InfoHash("11111111111111111111")
+	hash2 = chihaya.InfoHash("22222222222222222222")
+	hash3 = chihaya.InfoHash("33333333333333333333")
+
+	peers = []chihaya.Peer{
```

Comment:
I like to make test data like this the most readable as possible and then have a function that creates the right type.

For example this could be

```go
var []struct{
  seeder bool
  peerID string
  ip string
  port int32
}{
  {true, "-AZ3034-6wfG2wk6wWLc", "127.0.0.1", 5000},
}
```

You can find some example PeerIDs in [this file](https://github.com/chihaya/chihaya/blob/e2da9cdc3ebf5eec300773f9aaa894123ce3ba6a/pkg/clientid/client_id_test.go)